pagopa/io-web-profile
# [IOPID-728] : Adding Favicon and apple/android icons

## Short description
Added icons for Web, Apple devices and Android devices
## List of changes proposed in this pull request

- Web Favicon
- Apple Favicons
- Android Favicons

## How to test
yarn run build

check if icons are inside the folder **out/** :
- icons (android)
- apple-icons (apple) 
- favicon

then
yarn run start-static

**verify if icons refs are present inside head tag**
![image](https://github.com/pagopa/io-web-profile/assets/139151141/8a0fb003-7112-4248-b91d-adc9266cf1d1)
